### PR TITLE
first half of page looks ok in desktop mode but when changing size do…

### PR DESCRIPTION
**Pull request by DanAbsolon**
…esn't

respond well yet

### CODE REVIEW

File: danPortfolio.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+<body>
+	
+	<nav>
+		<div class="container-for-nav-buttons">
+			<a href="#about" class="three-nav-buttons">about</a>
+			<a href="#portfolio" class="three-nav-buttons">portfolio</a>
+			<a href="#contact" class="three-nav-buttons">contact</a>
+		</div>
+		<a href="#" class="download-cv">download CV</a>
```
Comment (robin-hodge) [inline danPortfolio.html]: make this link operational

File: danPortfolio.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	justify-content: space-evenly;
+	height: 100%;
+	padding: 75px;
+}
+
+.brief-intro {
+	padding: 25px;
+}
+
+.about {
```
Comment (robin-hodge) [inline danPortfolio.css]: adding flex-wrap: wrap will allow you to move picture from side to top/underneath

File: danPortfolio.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+.container-for-text-in-about p, h2 {
+	margin-bottom: 20px;
+}
+
+#photo-of-me {
+	margin-top: 20px;
+}
+
+
+.portfolio {
```
Comment (robin-hodge) [inline danPortfolio.css]: flex wrapping could help here

File: danPortfolio.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+			<p>Here are some projects in which I was involved.</p>
+			<div class="project-examples">
+				<h4>Project title</h4>
+				<img img src="https://cdn-developer-wp.arc.dev/wp-content/uploads/2022/06/coding-programming-project-ideas-1128x635.jpg" alt="image representing our project" width="100px" height="#" style="#">
+				<article class="project-description">In this project, we etc.</article>
+			</div>
+			<div class="project-examples">
+					<h4>Project title</h4>
+					<img img src="https://cdn-developer-wp.arc.dev/wp-content/uploads/2022/06/coding-programming-project-ideas-1128x635.jpg" alt="image representing our project" width="100px" height="#" style="#">
+					<article class="project-description">In this project, we etc.</article>
```
Comment (robin-hodge) [inline danPortfolio.html]: I think you want to make the text longer here, to help you correctly format this element in terms of text size

File: danPortfolio.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+				</div>
```
Comment (robin-hodge) [inline danPortfolio.html]: add 'git' and 'live' links here

File: danPortfolio.html [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+			<a href="#contact" class="three-nav-buttons">contact</a>
+		</div>
+		<a href="#" class="download-cv">download CV</a>
+	</nav>
+	
+	<section class="intro" id="section-with-windows-image">
+		<div class="box container-for-image-and-text-in-intro">
+			<img id="windows-image" src="https://images.adsttc.com/media/images/61e5/71ac/bcfa/3101/659e/e10c/newsletter/bdh-40.jpg?1642426848" alt="windows" width=100%>
+			<article class="brief-intro">
+				<h3>Hello there!</h3>
```
Comment (robin-hodge) [inline danPortfolio.html]: it is worth checking for correct use of H3 in line with best practice

File: danPortfolio.html [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+					<p>Outside coding, I enjoy playing the guitar, exercising, and reading.</p>  
+			</article>
+			<div class="container-for-image-in-about">
+				<img id="photo-of-me" img src="https://lh3.googleusercontent.com/a/AEdFTp6WY4Ej7I-rAKYjfmsA3DnIcXg3Ad3tTjwqu6sARQ=s576-p-no" alt="a photo of Dan" width="175px" height="#" style="#">
+			</div>
+		</div>
+	</section>
+
+	<section class="portfolio" id="portfolio">
+		<div class="box">
```
Comment (robin-hodge) [inline danPortfolio.html]: the naming of this class could be more semantic

File: danPortfolio.css [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+
+.box {
+	max-width: 50%;
+}
+
+a:hover { 
+  color: red;
+}
+
+.download-cv {
```
Comment (robin-hodge) [inline danPortfolio.css]: some of the formatting here could be imrpoved, as the border overlaps the text

File: danPortfolio.html [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+			<h2>Portfolio</h2>
+			<p>Here are some projects in which I was involved.</p>
+			<div class="project-examples">
```
Comment (robin-hodge) [inline danPortfolio.html]: some CSS styling here for flex/wrap etc.

File: danPortfolio.html [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+					<img img src="https://cdn-developer-wp.arc.dev/wp-content/uploads/2022/06/coding-programming-project-ideas-1128x635.jpg" alt="image representing our project" width="100px" height="#" style="#">
+					<article class="project-description">In this project, we etc.</article>
+				</div>
+			</div>
+	</section>
+
+	<section class="contact" id="contact">
+		<h2>Contact</h2>
+		<p>The best way to get in touch with me is via email, but if you prefer social media I have included those too.</p>
+		<a href="#">
```
Comment (robin-hodge) [inline danPortfolio.html]: add active links here

File: danPortfolio.html [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+			<img class="social-link" src="https://icon.signature.email/social/linkedin-circle-small-0077b5-FFFFFF.png" alt="linkedin-link" style="#">
+		</a>
+		<a href="#">
+			<img class="social-link" src="https://icon.signature.email/social/github-circle-small-24292e-FFFFFF.png" alt="github-link" style="#">
+		</a>
+	</section>
+
+	<footer>
+		<p>&copy 2023</p>
```
Comment (robin-hodge) [inline danPortfolio.html]: we talked about maybe including 'contact' in the footer

File: danPortfolio.html [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+			</div>
+		</div>
+	</section>
+
+	<section class="portfolio" id="portfolio">
+		<div class="box">
+			<h2>Portfolio</h2>
+			<p>Here are some projects in which I was involved.</p>
+			<div class="project-examples">
+				<h4>Project title</h4>
```
Comment (horsenelly) [inline danPortfolio.html:51]: some indentation needs to be checked, as is the case here

File: danPortfolio.html [hunk 13]
```diff
@@ -0,0 +1,88 @@
+<html>
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (robin-hodge) [inline danPortfolio.html]: too far indented

File: danPortfolio.html [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+		<div class="box">
+			<h2>Portfolio</h2>
+			<p>Here are some projects in which I was involved.</p>
+			<div class="project-examples">
+				<h4>Project title</h4>
+				<img img src="https://cdn-developer-wp.arc.dev/wp-content/uploads/2022/06/coding-programming-project-ideas-1128x635.jpg" alt="image representing our project" width="100px" height="#" style="#">
+				<article class="project-description">In this project, we etc.</article>
+			</div>
+			<div class="project-examples">
+					<h4>Project title</h4>
```
Comment (robin-hodge) [inline danPortfolio.html]: indented too far here

**Review by robin-hodge — CHANGES_REQUESTED** [review]
this is a request of the changes

File: danPortfolio.html (hunk 11 above)
Comment (DanAbsolon) [reply danPortfolio.html]: I reckon I'm just gonna have "back to top" and necessary copyright there

File: danPortfolio.html (hunk 1 above)
Comment (robin-hodge) [reply danPortfolio.html]: This still needs to be resolved!

**Review by robin-hodge — COMMENTED** [review]
Another look needs to be done here in terms of the formatting that changes at the top with the arrow spilling over. You seeem to have a workaround though!

**Pull request closed without merging**